### ISSUE TITLE
5.0 beta emits `const` keywords

### ISSUE THREAD

**Issue by aaronjensen** [styleguidist/react-styleguidist#295]
We use phantomjs to test that our styleguide loads. For some reason, after upgrading to 5.0.0, our styleguide bundle contains `const` declarators instead of `var` which don't work w/ phantom.

 I'm not sure what would have changed or how to track it down. Any ideas?

**Comment by sapegin** [comment]
I’ve tried to grep bundle of example styleguide and couldn‘t find any `const` so it shouldn‘t be from Styleguidist itself. Maybe some dependency was updated.

I‘ll be happy to help with debugging but I need the code ;-)

**Comment by aaronjensen** [comment]
Here's a repro: https://github.com/aaronjensen/react-styleguidist-const-repro

To repro:

```
$ yarn
$ yarn run build:styleguide
$ grep -e "const .=" priv/static/styleguide/build/bundle.js
```

The const is here: https://github.com/styleguidist/react-styleguidist/blob/next/loaders/utils/evalInContext.js#L18

**Comment by sapegin** [comment]
If it’s only in this function then it’s easy to fix. Thanks for the investigation!

**Comment by aaronjensen** [comment]
It also leaves consts in in the code examples, is there something that compiles those at runtime so they end up as var?

**Comment by sapegin** [comment]
I think the simples way would be to write examples with `var` ;-)

**Comment by aaronjensen** [comment]
hmm, but did something change from 4 to 5?

**Comment by sapegin** [comment]
For the file you mentioned earlier — yes, and it‘s easy to fix. For processing examples code: no, there‘s no processing.

**Comment by aaronjensen** [comment]
so in 4 it would have been eval'd w/ const? I wonder why my tests didn't complain about that...

**Comment by sapegin** [comment]
Because we run example code via Buble in a browser anyway. So fixing this one new `const` must be enough.

**Comment by sapegin** [comment]
Should be fixed in `react-styleguidist@5.0.0-beta.10`. Please try if you have time 🦄

**Comment by aaronjensen** [comment]
This is working now, thank you.
